thrimshim/thrimshim/main.py

```diff
@@ -155,6 +155,25 @@ def update_row(ident, new_row):
 	logging.info('Row {} updated to state {}'.format(ident, new_row['state']))
 	return ''
 
+@app.route('/thrimshim/reset/<ident>')
+def reset_row(ident):
+	"""Clear state and video_link columns and reset state to 'UNEDITED'."""
+	try:
```

Comment:
you can get flask to do this bit for you by making your route have `<ident:uuid>`.

Reply:
Annoyingly, I can't seem to pass uuid objects to psycopg2 so I need to turn them into strings

Reply:
Looks like you need to register it on startup: http://initd.org/psycopg/docs/extras.html?highlight=uuid#psycopg2.extras.register_uuid

I'm sure there's some annoying backwards-compatability reason for the default to be broken.